server/controllers/inventory/import/index.js

```diff
@@ -34,7 +34,9 @@ function downloadTemplate(req, res, next) {
  */
 function importInventories(req, res, next) {
   if (!req.files || req.files.length === 0) {
-    next(new BadRequest('Something broke', 'ERRORS.EVERYTHING_BAD'));
+    const errorDescription = 'Expected at least one file upload but did not receive any files.';
```

Comment:
This kind of error handling looks like it would be best handled by a middleware.  Could we change the `upload.middleware()` function, or create our own middleware, that simply throws an error if a route does not have `req.files` set?